tests/unit/test_utilities.inc

```diff
@@ -296,6 +296,18 @@ void print_matrix_complex_first_row_positive(const igraph_matrix_complex_t *matr
     igraph_matrix_complex_destroy(&copy);
 }
 
+void matrix_init_int(igraph_matrix_t *mat, int ncol, int nrow, int* elem) {
```

Comment:
Please specify in the function name whether the elements are in row-major or column-major order. (Something like `matrix_init_int_rowwise`).

Reply:
(Internally, igraph uses column-major storage, but this function interprets `elem` in row-major order, which is potentially confusing.)